scripts/gui/microbe_editor.mjs

```diff
@@ -310,6 +319,22 @@ function onExitToMenuClickedEditor(){
     }
 }
 
+function updateGuiButtons(isNucleusPresent){
+    if(!isNucleusPresent) {
+        $("#addMitochondrion").addClass("DisabledButton");
```

Comment:
This doesn't need jQuery, it should be used only when it makes something easier. See the other places where the classlist is changed.